recipes/nr.util/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "nr.util" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nr.util-{{ version }}.tar.gz
+  sha256: bc8767c3511b0bd21cc38b081a679c9ac07ded17699e7bb265977763bb257cd0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core >=1.0.0
+    - python >=3.7,<4.0
```

```suggestion
    - python >=3.7
```
Please remove this upper bound. When the time comes, we'll handle this internally.